src/client/sandbox/node/document/writer.js

```diff
@@ -190,14 +191,38 @@ export default class DocumentWriter {
         nativeMethods.insertBefore.call(elWithContent.parentNode, beginMarker, elWithContent);
     }
 
+    _getStartsWithClosingTagRegExp (tagName) {
+        tagName = tagName.toLowerCase();
+
+        if (!this.cachedRegExps[tagName]) {
+            const regExpStrParts = [tagName.charAt(tagName.length - 1), '?'];
```

Comment:
Move code, creating the regexp by tagName, to the separte function `createStartsWithClosingTagRegExp`